vendor/gmp.wake

```diff
@@ -13,49 +13,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package build_wake
 from wake import _
 from gcc_wake import _
 
 def gmp = match _
-  Pair "wasm-cpp14-release" _ = buildGmpWASM Unit
-  _ = Pass (pkg "gmp")
+    Pair "wasm-cpp14-release" _ = buildGmpWASM Unit
+    _ = Pass (pkg "gmp")
+
 
 def buildGmpWASM _ =
-  require Some emmake =
-    emmake
-  else
-    failWithError "emmake is not available"
+    require Some emmake = emmake
+    else failWithError "emmake is not available"
 
-  def emsdk =
-    replace `/[^/]*$` "" emmake
+    def emsdk = replace `/[^/]*$` "" emmake
 
-  def version = "6.2.1"
+    def version = "6.2.1"
 
-  require Pass buildDir = vendorBuildDir Unit
+    require Pass buildDir = vendorBuildDir Unit
 
-  def job =
-    """
+    def job =
+        """
       cd .build
       wget https://ftp.gnu.org/gnu/gmp/gmp-%{version}.tar.xz
       tar xmf gmp-%{version}.tar.xz
```

Comment:
How can we handle this?

Reply:
We can't really. There is no safe edit we can make inside of the `"""` that doesn't change the users code. In _most_ places this is fine but it _is_ a semantic change.

I think the best/correct option is to detect this in static analysis and emit a warning ala "the indentation of a multi-line string doesn't match the quote indentation, is this correct?"

Reply:
I learned a new thing. Did I miss this `"""` syntax somewhere in the documentation? Is it *just* a multiline string or is it something more special?